firebase-perf/src/main/java/com/google/firebase/perf/metrics/validator/PerfMetricValidator.java

```diff
@@ -135,33 +134,42 @@ public static String validateMetricName(@Nullable String str) {
   /**
    * Checks whether the given map entry fits key/value constraints for a Trace Attribute.
    *
-   * @param attribute A key/value pair for an Attribute
+   * @param key Key for the Attribute
+   * @param value Value for the Attribute
    * @return null if the entry can be used as an Attribute, if not, an error string explaining why
    *     it can't be used.
    */
   @Nullable
```

Comment:
Nit: This is no longer `@Nullable` since it's void :)

Reply:
Done!